### ISSUE TITLE
Use identifier literals in Proto to Ballerina conversion?

### ISSUE THREAD

**Issue by MaryamZi** [ballerina-platform/ballerina-lang#17764]
**Description:**
$title.

Currently if Ballerina keywords are field names, they seem to be prepended with the type. Instead we could use identifier literals and leave the field names as they are?

**Comment by MaryamZi** [comment]
For example 

```proto
message Event {
    string channel = 1;
}
```

is generated as 
```ballerina
public type Event record {|
    string string_channel; 

|};
```

Instead, we could generate as 
```ballerina
public type Event record {|
    string 'channel; 

|};
```

**Comment by daneshk** [comment]
This issue is fixed in #17988
